cli/fabric.c

```diff
@@ -676,16 +676,16 @@ static int hvd_body_print(struct switchtec_dev *dev,
 		}
 
 		if (!bound) {
-			printf("        Logical PID %hhu:\t\tUnbound\n", i);
+			printf("        Logical PID %d:\t\tUnbound\n", i);
```

Comment:
These changes from %hhu to %d looks like an unrelated bug fix.... It should be done in a different patch (same PR is fine).